tests/tests.rs

```diff
@@ -31,20 +31,27 @@ fn compare_generated_header(header: &PathBuf,
     if !expected.is_file() {
         let file_name = expected.file_name().unwrap().to_owned();
         expected.pop();
-
-        if cfg!(feature = "testing_only_libclang_4") {
-            expected.push("libclang-4");
-        } else if cfg!(feature = "testing_only_libclang_3_9") {
-            expected.push("libclang-3.9");
-        } else if cfg!(feature = "testing_only_libclang_3_8") {
-            expected.push("libclang-3.8");
+        
+        let version = clang_version();
+        match version.parsed {
```

Comment:
nit: You could do `let (maj, min) = version.parsed.expect("Couldn't parse libclang version!");`